### PR TITLE
Create termux-instructions.md

### PR DESCRIPTION
**Pull request by nmilosev**
Instructions for running on Android devices in Termux chroot

### CODE REVIEW

**Comment by encukou** [comment]
I'd rather fix the problem than document workarounds.

Why does pip not work? Can it be fixed instead?

Can you instead package elsa as a RPM?

Alternately, this could be served by the Flask server, I guess the changes/steps would be something like;
* Rename `site.py` to `fedoralovespython.py`
* Move the `from elsa import cli` line from the top to inside the `if __name__ == '__main__':` block
* Run with `FLASK_APP=fedoralovespython.py flask-3 run`

If that works for you, please pull request with the necessary changes.

File: termux-instructions.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+dnf update -y
+dnf install -y git python3-flask python3-markdown python3-yaml
+
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+git clone https://github.com/fedora-python/fedoralovespython.org
+cd fedoralovespython.org
+```
+- 3. You have to edit site.py (comment/remove these lines):
```
Comment (encukou) [inline termux-instructions.md:24]: Instructions to edit the source code should not be in documentation. Please change the source itself to make it work in all use cases.

**Comment by nmilosev** [comment]
> Can you instead package elsa as a RPM?

Probably the easiest solution.

`pip3` doesn't work, there is a weird bug where it creates folders with wrong umask, so it can't write to them.

I will look into changing this.

Thanks!

**Comment by nmilosev** [comment]
Added new commit, which allows for Flask server serving. Will test it out. Should work just fine on Termux without elsa.

**Comment by encukou** [comment]
I've squashed this and submitted as https://github.com/fedora-python/fedoralovespython.org/pull/27; let's move discussion there.

The comment should go in README. I removed it and noted it in in #8 for now.

**Comment by hroncok** [comment]
BTW use virtualenv with pip

**Comment by nmilosev** [comment]
Does using virtualenv fixes the pip issue? I'll have to try now.

EDIT: tried, doesn't work in termux How could we report this? dnf installs flask just fine for example (system-wide) but pip3 can't. I know we are having a very weird setup, but still. :(

**Pull request closed without merging**